### DISCUSSION TITLE
Unable to install companion app

### DISCUSSION THREAD

**Discussion by captmike1953** [aclap-dev/video-downloadhelper#1125] [Help and Q&A] [answered]
### Your question

I have an older MBP 15" (mid -2009) running OS 10.11.6 and I am unable to install the companion app for Videodownload helper.

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:78.0) Gecko/20100101 Firefox/78.0
coapp: no.

The error message is: The Installation failed.pdf

**Comment by paulrouget** [comment, marked as answer]
Sorry your MacOS version is too old. We don't support old version of MacOS.
